runtime/darwinia/src/migration.rs

```diff
@@ -72,6 +79,52 @@ fn migrate() -> frame_support::weights::Weight {
 		EVM::create_account(addr, REVERT_BYTECODE.to_vec());
 	}
 
+	let mut w = 32;
+
+	w += migration_helper::PalletCleaner {
+		name: b"BridgeKusamaGrandpa",
+		values: &[
+			b"RequestCount",
+			b"InitialHash",
+			b"BestFinalized",
+			b"ImportedHashesPointer",
+			b"CurrentAuthoritySet",
+			b"PalletOwner",
+			b"PalletOperatingMode",
+		],
+		maps: &[b"ImportedHashes", b"ImportedHeaders"],
+	}
+	.remove_all();
+	w += migration_helper::PalletCleaner {
+		name: b"BridgeKusamaParachain",
+		values: &[b"PalletOwner", b"PalletOperatingMode"],
+		maps: &[b"ParasInfo", b"ImportedParaHeads", b"ImportedParaHashes"],
+	}
+	.remove_all();
+	w += migration_helper::PalletCleaner {
+		name: b"BridgeCrabMessages",
+		values: &[b"PalletOwner", b"PalletOperatingMode"],
+		maps: &[b"InboundLanes", b"OutboundLanes", b"OutboundMessages"],
+	}
+	.remove_all();
+	w += migration_helper::PalletCleaner {
+		name: b"BridgeCrabMessages",
+		values: &[b"PalletOwner", b"PalletOperatingMode"],
+		maps: &[b"InboundLanes", b"OutboundLanes", b"OutboundMessages"],
+	}
+	.remove_all();
+	w += migration_helper::PalletCleaner {
+		name: b"CrabFeeMarket",
+		values: &[
+			b"Relayers",
+			b"AssignedRelayers",
+			b"CollateralSlashProtect",
+			b"AssignedRelayersNumber",
+		],
+		maps: &[b"Orders", b"RelayersMap"],
+	}
+	.remove_all();
+
 	// frame_support::weights::Weight::zero()
-	RuntimeBlockWeights::get().max_block
+	<Runtime as frame_system::Config>::DbWeight::get().reads_writes(0, w as _)
```

Comment:
I suggest changes this to the `max_block`, the currently mannual `w` missing the write operation for the `create_account` above. Also apply to the other runtimes.